Base.rte/AI/HumanAI.lua

```diff
@@ -1,15 +1,176 @@
-
 dofile("Base.rte/Constants.lua")
-require("AI/NativeHumanAI")  --dofile("Base.rte/Actors/AI/NativeHumanAI.lua")
+require("AI/NativeHumanAI")
 
 function Create(self)
-	self.AI = NativeHumanAI:Create(self)
+	self.AI = NativeHumanAI:Create(self);
+
+	-- You can turn features on and off here
+	self.armSway = true;
+	self.autoEquip = false;
+	self.alternativeGib = true;
+	self.visibleInventory = true;
+
+	self.pushStrength = 10;	-- Variable
+	self.tilt = 0.3;	-- Visible inventory item angle
+
+	self.lastAngle = self:GetAimAngle(false);
+
+	if self.armSway == true then
+		self.lastHandPos = {self.Pos, self.Pos};
+	end
+	
+	if self.alternativeGib == true then
+		self.detachLimit = self.GibWoundLimit;
+		self.GibWoundLimit = self.GibWoundLimit * 1.5;
+	end
+end
+
+function Update(self)
+
+	self.ctrl = self:GetController();
+	
+	-- Detach limbs instead of regular gibbing
+	if self.alternativeGib == true then
+		if self.WoundCount > self.detachLimit then
+			self.detachLimit = self.WoundCount + 1;
+			local parts = {self.BGArm, self.BGLeg, self.FGArm, self.FGLeg, self.Head};	-- Piority order
+			local mostWounds = -1;
+			local detachLimb;
+			-- Pick the limb with most wounds and detach it
+			for i = 1, #parts do
+				local limb = parts[i];
+				if limb and limb.WoundCount > mostWounds then
+					detachLimb = limb;
+					mostWounds = limb.WoundCount;
+				end
+			end
+			if detachLimb then
+				detachLimb.JointStrength = -1;
+			end
+		end
+	end
+	
+	-- Equip a weapon automatically if the one held is destroyed
+	if self.autoEquip == true then
+		if self:IsPlayerControlled() and self.EquippedItem == nil and self.InventorySize > 0 then
+			self:EquipFirearm(true);
+		end
+	end
+	
+	-- Flail around if moving mouse too fast
+	if self.ctrl:IsMouseControlled() then
+
+		local mouseVec = Vector(self.ctrl.MouseMovement.X, self.ctrl.MouseMovement.Y):SetMagnitude(math.sqrt(self.ctrl.MouseMovement.Magnitude));
+		local ang = self.lastAngle - self:GetAimAngle(false);
+
+		self.AngularVel = self.AngularVel - (2 * ang * self.FlipFactor + mouseVec.Y * self.FlipFactor / 10) / math.sqrt(math.abs(self.AngularVel) + 1);
```

Comment:
There should be no spaces before or after `/`s. See the [style guide](https://github.com/cortex-command-community/Cortex-Command-Community-Project-Data/wiki/Lua-Style-Guide), section 6.
This applies in other places too, but I forgot to mention it when I was going through everything the first time. Correct it where you see it but don't worry too much if you miss some.

Reply:
Fuck, oh well, I'll look out for them.